### DISCUSSION TITLE
Is it possible use it for free?

### DISCUSSION THREAD

**Discussion by diogobratti** [itsToggle/plex_debrid#444] [Q&A] [answered]
I was following this guide [linux-arm64-server-setup](https://github.com/itsToggle/plex_debrid/wiki/Setup-Guides#linux-arm64-server-setup) and I was not able to create a debrid API Key. I suppouse I have to pay for it. Am I doing something wrong?

**Comment by itsToggle** [comment, marked as answer]
No, debrid services are paid services. I'd recommend realdebrid, it's 3$ a month which is about as cheap as it gets
